lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2301,7 +2302,13 @@ def delta(self, other):
                 if different:
                     break
 
+            # Need to update hatching if we also updated fillcolor
+            if params == ('_hatch',) and fill_performed:
```

Comment:
To check the logic, it only matter if the _last_ command was filled so the fact that the first time through is always False is ok?